runners/pvc_autoresizer.go

```diff
@@ -274,6 +281,27 @@ func convertSizeInBytes(valStr string, capacity int64, defaultVal string) (int64
 	return val, nil
 }
 
+func convertSize(valStr string, capacity int64, defaultVal string) (int64, error) {
+	if len(valStr) == 0 {
+		valStr = defaultVal
+	}
+
+	if !strings.HasSuffix(valStr, "%") {
+		return 0, fmt.Errorf("annotation value should be in percent notation: %s", valStr)
+	}
+
+	rate, err := strconv.ParseFloat(strings.TrimRight(valStr, "%"), 64)
+	if err != nil {
+		return 0, err
+	}
+	if rate < 0 {
+		return 0, fmt.Errorf("annotation value should be positive: %s", valStr)
+	}
```

Comment:
Please forgive my ignorance, but is the check `rate > 100` unnecessary?

Reply:
Sorry, I fixed it.
https://github.com/topolvm/pvc-autoresizer/pull/65/commits/db297872dd832fd7aa9c1ad1593eb67d52d9c691